Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'rails', '~> 7.1.2'
 
 gem 'bootsnap', require: false
 gem 'cssbundling-rails'
+gem 'dotenv-rails', '~> 2.8', '>= 2.8.1', groups: [:development, :test]
```

Comment:
doing this is the same as adding the gem directly in the development and test group. Can you move it over to the appropiate group?
```ruby
group :development, :test do
  gem 'dotenv-rails', '~> 2.8', '>= 2.8.1'
end
```